library/src/scala/implicits/Not.scala

```diff
@@ -0,0 +1,9 @@
+package scala.implicits
+
+final class Not[+T] private ()
```

Comment:
Some documentation would be nice :).

Reply:
Sure :-) As well as spec the whole thing.